extensions/amp-carousel/0.1/scrollable-carousel.js

```diff
@@ -126,6 +134,45 @@ export class AmpScrollableCarousel extends BaseCarousel {
     }
   }
 
+  /**
+   * Scrolls to the slide at the given slide index.
+   * @param {*} value
+   * @private
+   */
+  goToSlide_(value) {
+    const index = parseInt(value, 10);
```

Comment:
whole function should become a measure/mutate, where `getPosForSlideIndex_` is called in the measure block and start of animation is in the mutate block.